### ISSUE TITLE
Broken link on 18F Accessibility Checklist page

### ISSUE THREAD

**Issue by amycole501** [18F/guides#316]
Broken link for "plug-ins" (bottom of the "C" list)
https://accessibility.18f.gov/checklist/

[Required plugins are linked on the page](https://accessibility.18f.gov/plugins/)

**Comment by christophermather** [comment]
We encountered this issue in migrating the Guides to a new repo, as well. At the time, we were unable to locate a replacement link and decided to remove the link entirely: https://guides.18f.gov/accessibility/checklist/

<img width="407" alt="image" src="https://github.com/18F/guides/assets/97245800/88c1d977-c789-4dcc-a5d5-c94f807853ee">

This needs some investigation, maybe starting w/ the current code owner for Accessibility (@jasnakai).

**Comment by amycole501** [comment]
Sounds good thanks for following up

**Comment by christophermather** [comment]
@jasnakai - when you have a second, could you take a look at this issue? The basics:

- On the [Checklist](https://guides.18f.gov/accessibility/checklist/) page, item number 7 under `C - Minor` is "Required plugins are linked on the page"
- On the old guide, the text `Required plugins` used to link to `https://accessibility.18f.gov/plugins/`
- But, as Amy has noted, that link has been broken since at least September
- We encountered this issue while replatforming, as well
- We investigated and couldn't find a `/plugins/` page
- We decided to simply remove the link to allow us to proceed with replatforming
- Now that we're replatformed, we need to fix

Two options:

1. Investigate and figure out if a `/plugins/` page existed and can be restored, and/or
2. Simply remove item 7 from the `C - Minor` checklist

**Comment by jasnakai** [comment]
Will do. Thank yoU!

**Comment by jasnakai** [comment]
I found the information in the GSA Accessibility Statement (https://www.gsa.gov/website-information/accessibility-statement).
I will update the link to point to this source.

**Comment by jasnakai** [comment]
solved with this PR: [Fixed broken link on 18F Accessibility Checklist page #548](https://github.com/18F/guides/pull/548)
